index.html

```diff
@@ -8,8 +8,19 @@
 	<link rel="stylesheet" href="index.css">
 </head>
 <body>
-
-	<h1>NK Flohmarkt am Kranoldplatz coming soon...</h1>
-
+	<div class="Grid">
+		<div class="Grid-socialList">
+			<a class="Grid-socialList-icon" href="www.facebook.com"><img src="./assets/facebook.png" alt="broken"></a>
+			<a class="Grid-socialList-icon" href="www.twitter.com"><img src="./assets/twitter.png" alt="broken"></a>
+			<a class="Grid-socialList-icon" href="www.instagram.com"><img src="./assets/instagram.png" alt="broken"></a>
+		</div>
+		<div class="Grid-logoWrapper"><img src="./assets/logo.png" alt="broken"></div>
```

Comment:
set "NK Flohmarkt am Kranoldplatz" as the alt attribute of the img. I would also wrap the image in an `h1` to indicate it's the title of the page.

Reply:
Do you mean??

<div class="Grid-logoWrapper">
    <h1>
        <img src="./assets/logo.png" alt="broken">
    </h1>
</div>

Reply:
exactly, just with a useful `alt` attribute instead of "broken"